client/src/GlobalStyle.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { Global, css } from "@emotion/core";
+import colors from "../src/utils/colors";
+
+export default function GlobalStyle() {
+  return (
+    <Global
+      styles={css`
+        *,
+        *:before,
+        *:after {
+          box-sizing: border-box;
+        }
+        body {
+          height: 100vh;
+          font-size: 0.9rem;
+          margin: 0;
+          font-family: "Montserrat", sans-serif;
+          color: ${colors.primaryText};
+          background-color: ${colors.background};
+          &:after {
```

Comment:
This looks a bit like a workaround. I think it is because of the opacity of your background that you use `:after` to create a different element that has opacity and the background.

However, you are also able to download the image yourself, edit it with opacity 30% in an image editing tool and set it as a normal background for the body without the pseudo selector `:after`.

Reply:
Good idea! I'll do this in another pull request when I create the image folder.